docs/components/configMaps.md

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.configMaps`
+
+This grade provides the configuration maps for the gamepad in the form of component's model data.
```

Comment:
We call this type of component a "mix in", i.e. you mix it in with one or more grades that provide actual functionality.  You don't have to mention that, but you should mention the grades it's meant to be mixed with.  If you want to handle that when writing those docs, leave a comment with a TODO, please.

Reply:
Also, you should rework "in the form of component's model data".  One option is to just remove it, you cover the model data parts already in later paragraphs.